### PR TITLE
schemas added for post comments and post likes

### PR DESCRIPTION
**Pull request by george-h-h**
## Before you start

1. Have you run `git pull origin main` on your branch to bring in the latest changes?
2. Is there an issue on the repository for this request? If not, please create one.
3. Have you followed the correct workflow in the [team documentation](https://docs.google.com/document/d/13yi709ax4nmfTG9J0F57yEFpYRH7fdtczwvVJCrNIsM/edit#)?
4. Conflict resolution to be dealt with **on GitHub**.

## What do these code changes do?

Database changes, schemas added to record comments and likes on posts.

## How do these code changes affect the application?

No front end changes... yet, so this is simply data processing and handling.

## Checklist before requesting a review

- [X] I have performed a self-review of my code
- [X] I have added thorough tests.
- [X] I have created a PR ticket on the [Trello board](https://trello.com/b/P8lEsvZe/acebook).
- [X] I will link the PR to the Trello card once created.
- [X] I will send the PR in the Slack channel and ping @here.

### CODE REVIEW

**Comment by george-h-h** [comment]
[New schemas for post comments and likes](https://trello.com/c/Dc0Z0cvm/79-new-schemas-for-post-comments-and-likes)

File: models/post_like.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now(),
+  },
```
Comment (PKilgarriff) [inline models/post_like.js:17]: Don't know if this would be needed for likes?

**Review by PKilgarriff — APPROVED** [review]
Happy with that George! My only comment would be whether a like needs a created_at field, because I can't think when we would use that (unless it's planned for notifications or similar?)

File: models/post_like.js (hunk 1 above)
Comment (george-h-h) [reply models/post_like.js:17]: You are right. They wouldn't be required. My bad.

**Review by heykathl — APPROVED** [review]
Great stuff George!!

**Pull request merged**